functions/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as functions from "firebase-functions";
+import * as firebaseAdmin from "firebase-admin";
+import * as corsLib from "cors";
+import { getAddress } from "ethers/lib/utils";
+
+const admin = firebaseAdmin.initializeApp();
+
+const cors = corsLib({
```

Comment:
can we add a comment on why this is needed?  Does it affect the entire app?

Reply:
Commented in the code. It only affects the functions that use it, and it's necessary to make HTTP calls to a function